src/CodeGen_C.h

```diff
@@ -115,6 +115,9 @@ class CodeGen_C : public IRPrinter {
     /** Emit a version of a string that is a valid identifier in C (. is replaced with _) */
     virtual std::string print_name(const std::string &);
 
+    /** Add platform speciefic headers. Needed for Xtensa */
```

Comment:
I don't think "Needed for Xtensa" part of the comment is needed, because this might be used on other platforms too.